Description: remove SONAME
 libsocket_wrapper is a preloadable library.
 We do not support directly linking to it, so it doesn't need the SONAME.
 Let's remove it.
Author: Jakub Wilk <jwilk@debian.org>
Last-Update: 2020-04-16

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -16,19 +16,6 @@ project(socket_wrapper VERSION 1.2.4 LAN
 # global needed variables
 set(APPLICATION_NAME ${PROJECT_NAME})
 
-# SOVERSION scheme: MAJOR.MINOR.PATCH
-#   If there was an incompatible interface change:
-#     Increment MAJOR. Set MINOR and PATCH to 0
-#   If there was a compatible interface change:
-#     Increment MINOR. Set PATCH to 0
-#   If the source code was changed, but there were no interface changes:
-#     Increment PATCH.
-set(LIBRARY_VERSION_MAJOR 0)
-set(LIBRARY_VERSION_MINOR 1)
-set(LIBRARY_VERSION_PATCH 15)
-set(LIBRARY_VERSION "${LIBRARY_VERSION_MAJOR}.${LIBRARY_VERSION_MINOR}.${LIBRARY_VERSION_PATCH}")
-set(LIBRARY_SOVERSION ${LIBRARY_VERSION_MAJOR})
-
 # add definitions
 include(DefinePlatformDefaults)
 include(DefineOptions.cmake)
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -12,8 +12,7 @@ target_link_libraries(socket_wrapper ${S
 
 set_target_properties(socket_wrapper
                       PROPERTIES
-                          VERSION ${LIBRARY_VERSION}
-                          SOVERSION ${LIBRARY_SOVERSION})
+                        NO_SONAME ON)
 if (DEFINED DEFAULT_LINK_FLAGS)
     set_target_properties(socket_wrapper
                           PROPERTIES
